addons/website/static/src/scss/website.scss

```
@@ -1246,6 +1257,59 @@ header {
     }
 }
 
+// Offcanvas menu on mobile or offcanvas themed header override
+.o_mobile_or_offcanvas {
+    .o_mega_nav, .o_mega_menu {
+        max-width: $offcanvas-horizontal-width;
+        transition: transform .3s ease-in-out, visibility .3s ease-in-out;
+        transform: translateX(var(--FakeOffcanvas-animation, 100%));
+    }
+
+    .o_mega_menu.show, .o_mega_menu.show ~ .o_mega_nav {
+        visibility: visible !important;
+        transform: translateX(0%);
+        transition: transform .3s ease-in-out, visibility .3s ease-in-out;
+    }
+
+    .o_mega_nav {
+        z-index: 1001;
+        height: $o_mega_nav_height;
+        padding-left: $grid-gutter-width / 2;
```

Convention: we normally multiply ;) 
```suggestion
        padding-left: $grid-gutter-width * .5;
```